packages/yarnpkg-core/sources/httpUtils.ts

```diff
@@ -100,31 +122,18 @@ export async function request(target: string, body: Body, {configuration, header
   const socketTimeout = configuration.get(`httpTimeout`);
   const retry = configuration.get(`httpRetry`);
   const rejectUnauthorized = configuration.get(`enableStrictSsl`);
-  const globalCaFilePath = configuration.get(`caFilePath`);
+  const caFilePath = networkConfig?.config.get(`caFilePath`) ?? configuration.get(`caFilePath`);
```

Comment:
Same thing, the global `caFilePath` setting should be "resolved" inside the `networkConfig` (similar to what we do in `getAuthConfiguration`, for instance).